lib/metric_fu/metrics/flog/generator.rb

```diff
@@ -1,5 +1,6 @@
 require "pathname"
 require "optparse"
+require "path_expander"
```

Comment:
I think, if we are using path_expander directly it's better add it as dependency as you mentioned. Because if *flog* stop using `path_expander`, we could have an issue here...

What you think?